contrib/completions/bash/kubectl

```diff
@@ -288,6 +288,7 @@ _kubectl_get()
     must_have_one_noun=()
     must_have_one_noun+=("componentstatus")
     must_have_one_noun+=("daemonset")
+    must_have_one_noun+=("dedicatedmachine")
```

Comment:
We don't use the term "machine" anywhere in Kubernetes. Let's not introduce another term in addition to minion and host. Node is the appropriate term.